src/EnergyPlus/PluginManager.cc

```diff
@@ -1272,6 +1272,9 @@ namespace PluginManagement {
                 if (functionName == this->sHookBeginNewEnvironment) {
                     this->bHasBeginNewEnvironment = true;
                     this->pBeginNewEnvironment = (*EP_PyString_FromString)(functionName.c_str());
+                } else if (functionName == this->sHookBeginZoneTimestepBeforeSetCurrentWeather) {
+                    this->bHasBeginZoneTimestepBeforeSetCurrentWeather = true;
+                    this->pBeginZoneTimestepBeforeSetCurrentWeather = (*EP_PyString_FromString)(functionName.c_str());
```

Comment:
Attempted to add the new calling point on thePython side. Untested.

Reply:
I'll test.  Thanks for putting this in place.